packages/react-label/MIGRATION.md

```diff
@@ -0,0 +1,25 @@
+# Label Migration
+
+## STATUS: WIP
+
+This Migration guide is a work in progress and is not yet ready for use.
+
+## Migration from v8
+
+- `Label`
+  - `componentRef` => Not supported.
+  - `disabled` => `disabled`.
+  - `required` => `required`.
+
+## Migration from v0
+
+The converged API does not support many of the features offered in v0. Some could potentially by addressed by already made slots or by adding either the additional features or more slots if needed.
+
+- `Label`
+  - `circular` => Not supported. Consider using `Badge` component.
+  - `color` => Not supported. Consider using `Badge` component.
+  - `content` => The child of `Label`.
+  - `design` => Not applicable.
+  - `fluid` => Not supported. Use CSS styling such as flex-grow.
+  - `icon`, `iconPosition` => Not supported. Consider using `Badge` component or add as a child.
```

Comment:
This might be another use case for the info tooltip slot.

Reply:
Since in the meeting earlier we agreed to remove the info slot for now, I think it's fine to leave as is. If we were to add the info slot again in the future, then we should consider adding a suggestion to use the info slot.